keyserver/coniksserver/coniksserver.go

```diff
@@ -0,0 +1,148 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/signal"
+	"path"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+	"github.com/coniks-sys/coniks-go/crypto/sign"
+	"github.com/coniks-sys/coniks-go/crypto/vrf"
+	"github.com/coniks-sys/coniks-go/keyserver"
+	"github.com/coniks-sys/coniks-go/utils"
+)
+
+type boolStringFlag struct {
+	set   bool
+	value string
+}
+
+func (sf *boolStringFlag) Set(x string) error {
+	sf.set = true
+	if x != "true" {
+		sf.value = x
+	}
+	return nil
+}
+
+func (sf *boolStringFlag) String() string {
+	return sf.value
+}
+
+func (sf *boolStringFlag) IsBoolFlag() bool {
+	return true
+}
+
+func main() {
+	genkey := boolStringFlag{false, "sign,vrf"}
+	flag.Var(&genkey, "genkey", "Generate signing/VRF key, e.g. -genkey=sign,vrf")
+	genConfig := flag.Bool("genconfig", false, "Generate server configurations file")
+	dir := flag.String("dir", ".", "Directory location for storing generated files")
+	confPath := flag.String("config", "config.toml", "Path to the configuration file")
+
+	flag.Parse()
+
+	switch {
+	case *genConfig:
+		mkConfig(*dir)
+		fallthrough
```

Comment:
Hmm, I have assigned a default value for `genkey` (https://github.com/coniks-sys/coniks-go/blob/cdaf4bf18b7f801149e6766f7239a301ec0c2365/keyserver/coniksserver/coniksserver.go#L42) and changed the switch a little bit. I think it's ok but not the best solution :(.